### DISCUSSION TITLE
After add custom location status offline

### DISCUSSION THREAD

**Discussion by baxranoff** [NginxProxyManager/nginx-proxy-manager#3507] [General]
After adding a custom location, the status will change to offline.

**Comment by 150170410** [comment]
seeing the same issue

I want configure below local system 

IP/site1  point to IP:3001 service, it does not work

**Comment by RealChaos23** [comment]
> After adding a custom location, the status will change to offline.

Try this solution (from @draedric7) ==> [here](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3474#issuecomment-1902790528)
